.github/workflows/publish.yml

```diff
@@ -29,7 +33,22 @@ jobs:
           make contributors
           make models
 
+      - name: Test
```

Comment:
Is this step supposed to be called test?

Reply:
Yes, because this is here to test that the site can build.